### PR TITLE
AGIOS-558 bump version for 3.0.0 release

### PR DESCRIPTION
**Pull request by jcesarmobile**
removing the `-swift` as they are different repos now

### CODE REVIEW

File: AeroGearPush-Swift.podspec [hunk 1]
```diff
@@ -1,11 +1,11 @@
 Pod::Spec.new do |s|
   s.name         = "AeroGearPush-Swift"
```
Comment (danielpassos) [inline AeroGearPush-Swift.podspec]: Still with `-swift` :joy:

**Comment by danielpassos** [comment]
@jcesarmobile Let's also rename `AeroGearPush-Swift.podspec` to `AeroGearPush.podspec`. Wdyt?

**Comment by jcesarmobile** [comment]
As we are still distributing the Objective-C one, better keep the -Swift in CocoaPods related names, I just removed it from the tag as now we will only have a tag from each repo

**Comment by danielpassos** [comment]
Gotcha! Sorry, my bad!

**Pull request merged**